icu4c/source/data/unidata/generate.sh

```diff
@@ -49,6 +49,5 @@ bazelisk run //tools/unicode/c/genprops $ICU_SRC/icu4c
 bazelisk run //tools/unicode/c/genuca -- --hanOrder implicit $ICU_SRC/icu4c
 bazelisk run //tools/unicode/c/genuca -- --hanOrder radical-stroke $ICU_SRC/icu4c
 # Also generate the ICU4X versions
-# TODO: Currently fails with early Unicode 16.0 FractionalUCA.txt.
-# bazelisk run //tools/unicode/c/genuca -- --icu4x --hanOrder implicit $ICU_SRC/icu4c
-# bazelisk run //tools/unicode/c/genuca -- --icu4x --hanOrder radical-stroke $ICU_SRC/icu4c
+bazelisk run //tools/unicode/c/genuca -- --icu4x --hanOrder implicit $ICU_SRC/icu4c
```

Comment:
Please also restore the "rm" lines above to their normal state (before my PR #2930).

Reply:
Uncommented on `rm` line.